src/xdocs/index.xml.vm

```diff
@@ -69,24 +69,6 @@
 
     </section>
 
-    <section name="Previous Version Documentation">
-      <p>
-        Current site contains documentation only for latest, released version.
```

Comment:
hmm, this section is good. It is required to let people know how to see documentation for old versions.

Reply:
It is now intuitive, i.e. keep scrolling to bottom of release information and see link to last interval, visit that link, repeat.

Reply:
lets move discussion on this section out of this PR.
I do not understand your point, more details are required. I do not want to block this pr due to this section

Reply:
Section is restored, issue created at https://github.com/checkstyle/checkstyle/issues/11754